### PR TITLE
Wochenmarkt Usingen added

### PR DESCRIPTION
**Pull request by telegnom**
Adds "Wochenmarkt Usingen" to the map.

### CODE REVIEW

**Comment by johnjohndoe** [comment]
@telegnom Please also update the README.

**Comment by telegnom** [comment]
Updated the README as required.

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ in February 2016.
 |[Trier][trier-wikipedia]|[City of Trier][trier-markets]|
 |[Tübingen][tübingen-wikipedia]|[Wochenmarkt - Universitätsstadt Tübingen][tübingen-markets]|
 |[Ulm][ulm-wikipedia]|[City of Ulm][ulm-markets]|
+|[Usingen][usingen-wikipedia]|[City of Usingen][usingen-markets]|
```
Comment (johnjohndoe) [inline README.md]: You are missing the links here as you can see when you scroll down the README here: https://github.com/telegnom/wo-ist-markt.github.io

File: cities/usingen.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                "title": "Usinger Wochenmarkt",
+                "location": "Alter Marktplatz, Usingen",
+                "opening_hours": "Fr 08:00-18:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Usingen",
```
Comment (johnjohndoe) [inline cities/usingen.json]: Please note that the [contribution guide](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/CONTRIBUTING.md) recommends to add a license and an update date here, e.g. `"title": "Stadt Berlin, CC BY 3.0 DE, aktualisiert am 03.05.2019",`

**Comment by telegnom** [comment]
So i finally think I fixed everything. In the future I'll create a separate branch for every contribution. I promise. This one was painful, but I at least learned something. And thanks for your patience.

**Comment by johnjohndoe** [comment]
I rebased your branch and merged here: 75888f57d50ec5fcf17b3ede05d0391594895b70.
Thank you for adding Usingen. :+1:

**Pull request closed without merging**